ext/curl/curl_string_file.stub.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * @generate-function-entries
```

Comment:
You can now get rid of `@generate-function-entries`

Reply:
Got it, thank you.